### PR TITLE
fix(amazonq): remove single quotes from messages

### PR DESCRIPTION
**Pull request by dhasani23**
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

When using the `message` function on a String, it is supposed to replace the `{0}` in the String with a provided value, however this does not work when the String contains a single quote in it (such as `couldn't`). 

We could escape this, but our PM does not want contractions in messages anyway, so solving this by removing the single quotes.

## Checklist
- [X] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties [hunk 1]
```diff
@@ -541,9 +541,9 @@ codemodernizer.chat.message.button.view_summary=View summary
 codemodernizer.chat.message.error_request=Request failed
 codemodernizer.chat.message.follow_up.new_transformation=Start a new transformation
 codemodernizer.chat.message.local_build_begin=I'm building your module. This can take up to 10 minutes, depending on the size of your module.
-codemodernizer.chat.message.local_build_failed=Sorry, I couldn't run the Maven clean install command to build your module.
```
Comment (mk-fan) [inline plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:544]: Is there no way to escape the single quote character?

Comment (damntrecky) [reply plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:544]: Have we asked the toolkit team about this. Usually for JS/TS you have string escape delimiters for special characters.


-    \\ – Backslash
 -   \' – Single quote
   -  \" – Double quote
    - \n – Newline
-    \r – Carriage return
  -  \t – Tab

Comment (damntrecky) [reply plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:544]: Have we asked the toolkit team about this. Usually for JS/TS you have string escape delimiters for special characters.


-    \\ – Backslash
-   \' – Single quote
   -  \" – Double quote
    - \n – Newline
-    \r – Carriage return
  -  \t – Tab

Comment (mk-fan) [reply plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:544]: https://stackoverflow.com/questions/4449639/apostrophe-doesnt-get-translated-properly-when-placed-in-a-resource-bundle

Comment (dhasani23) [reply plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties:544]: Spoke to PM and we don't want contractions in our messages anyway. So `could not` is preferred over `couldn't`.

**Review by manodnyab — APPROVED** [review]

**Pull request merged**